tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -116,6 +110,9 @@ AGENT_SOURCE_BIN    Path to the directory of agent binary.
 DISTRO_REPO         Use host repositories to install guest packages.
                     Default value: <not set>
 
+DOCKER_RUNTIME      Docker runtime to use when USE_DOCKER is set.
+                    Default value: runc
+
```

Comment:
You're just shuffling this one around, right?  As it's not exactly related to this patch, I'd put it in a different patch (but part of the same series).